### ISSUE TITLE
AUR build fails

### ISSUE THREAD

**Issue by SantoJambit** [lunaryorn/gnome-search-providers-vscode#8]
I'm getting this error during AUR build:
```console
install: cannot stat 'target/release/gnome-search-providers-vscode': No such file or directory
```

**Comment by lunaryorn** [comment]
It works for me in a clean chroot. I'm sorry but I can't really help you.

**Comment by SantoJambit** [comment]
@lunaryorn Can you tell me, how the makefile is called?
The log seems to suggest, that the build step happens after or (in parallel) during the install step:

```console
==> Starting package()...
cargo build --release --locked
install -Dm644 -t /var/tmp/pamac-build-.../gnome-search-providers-vscode/pkg/gnome-search-providers-vscode//usr/share/gnome-shell/search-providers providers/de.swsnr.searchprovider.vscode.arch.code-oss.ini providers/de.swsnr.searchprovider.vscode.arch.visual-studio-code.ini
install -Dm755 -t /var/tmp/pamac-build-.../gnome-search-providers-vscode/pkg/gnome-search-providers-vscode//usr/lib/gnome-search-providers-vscode/ target/release/gnome-search-providers-vscode
install: cannot stat 'target/release/gnome-search-providers-vscode': No such file or directory
make: *** [Makefile:20: install] Error 1
make: *** Waiting for unfinished jobs....
   Compiling proc-macro2 v1.0.26
   Compiling unicode-xid v0.2.1
...
   Compiling gnome-search-providers-vscode v1.1.0 (/var/tmp/pamac-build-.../gnome-search-providers-vscode/src/gnome-search-providers-vscode-1.1.0)
    Finished release [optimized] target(s) in 2m 28s
mkdir -p target/dbus-1 target/systemd
sed "s:{PREFIX}:/usr:g" "dbus-1/de.swsnr.searchprovider.VSCode.service" > "target/dbus-1/de.swsnr.searchprovider.VSCode.service"
sed "s:{PREFIX}:/usr:g" "systemd/de.swsnr.searchprovider.VSCode.service" > "target/systemd/de.swsnr.searchprovider.VSCode.service"
```

I'm on manjaro here if that's relevant.

**Comment by lunaryorn** [comment]
Oh, you've got -j in Makeflags?  

The makefile lacks a dependency from install to build, and with parallelism make would run both in parallel.  I don't set Makeflags for makepkg, so I didn't notice.

I'm sorry, I'll fix this.  Meanwhile you can change the PKGBUILD to run make build and make install in two separate commands.

**Comment by SantoJambit** [comment]
I never configured anything like it. Must be the default behavior of Manjaro. I know how to install from source. I just wanted to set it up via AUR, so it gets updates automatically, so I'll wait for the next release.

**Comment by SantoJambit** [comment]
I just checked /etc/makepkg.conf:
```
#-- Make Flags: change this for DistCC/SMP systems
MAKEFLAGS="-j$(($(nproc)+1))"
```

As said, I did not edit this file at all, so it must be the default behavior.

**Comment by lunaryorn** [comment]
@SantoJambit I published 1.1.1 which should fix this issue.
